server/dataone_upload.py

```diff
@@ -156,6 +164,41 @@ def create_upload_object_metadata(client, file_object):
     return pid
 
 
+def create_upload_remote_file(client, globus_files, user):
+    """
+    Creates and uploads a file that describes files that exist on external
```

Comment:
Any chance Globus pre-computes a checksum for us so we don't have to (1) upload the file to dataone or (2) compute the md5 locally first?

Reply:
Checking out the documentation, I don't see any functionality for it (confirmed in the dev call).
https://docs.globus.org/api/transfer/file_operations/